src/main/kotlin/com/anaplan/engineering/vdmgradleplugin/TypeChecker.kt

```diff
@@ -50,25 +50,29 @@ open class VdmTypeCheckTask(private val includeTests: Boolean) : OvertureTask()
         @InputFiles
         get() = project.locateAllSpecifications(dialect, includeTests)
 
-    val generatedLibFile: File
+    // Only needed to ensure caching of the Gradle task
+    val statusFile: File
```

Comment:
Ideally I would prefer to store a list of warnings and errors, but Overture/VDMJ does not expose those directly. We could simply save the logs of the parser and type checker, but I couldn't find a straightforward `tee` equivalent in Kotlin.

Reply:
My comment here got lost, see comment on L64